test/functional/e2e/user-profile-page-object.ts

```diff
@@ -0,0 +1,178 @@
+import puppeteer from 'puppeteer';
```

Comment:
maybe in general the folder structure to use could be `e2e/user/<user tests>`. or even broken down into several other folder related to the user, like authorization, profile, ecoverse, etc.